benchopt/plotting/plot_objective_curve.py

```diff
@@ -76,28 +128,42 @@ def plot_objective_curve(df, obj_col='objective_value', plotly=False,
             plt.text(0.5, 0.5, "Not Available")
         return fig
 
+    # use 2nd and 8th decile for now
+    q_min, q_max = 0.25, 0.75
     for i, solver_name in enumerate(solver_names):
         df_ = df[df['solver_name'] == solver_name]
-        curve = df_.groupby('stop_val').median()
 
-        q1 = df_.groupby('stop_val')['time'].quantile(.1)
-        q9 = df_.groupby('stop_val')['time'].quantile(.9)
+        if iteration:
+            curve_t = (
+                df_.groupby('stop_val')[obj_col]
+                .quantile([q_min, 0.5, q_max]).unstack()
+            )
+        else:
+            curve_t = _get_curve_interp(df_, obj_col, q_min=q_min, q_max=q_max)
 
-        fill_between_x(
-            fig, curve['time'], q1, q9, curve[obj_col], color=CMAP(i % CMAP.N),
-            marker=markers[i % len(markers)], label=solver_name, plotly=plotly
+        fill_between_y(
```

Comment:
I am not sure I prefer the interpolated plot. I understand it is weird that the curves goes back in time, but it is easily explained by variations in computational time. With the interpolation, the curves look like the solver is not always improving over iterations, and the explanation is more convoluted.